mnist/download_mnist.py

```diff
@@ -0,0 +1,5 @@
+import urllib.request, os
+filepath = os.path.dirname(os.path.abspath(__file__))
+if not os.path.exists(os.path.join(filepath,"mnist.h5")):
+    urllib.request.urlretrieve("https://sites.google.com/site/charleskennethfisher/home/programs-and-data/mnist.h5?attredirects=0&d=1",
```

Comment:
Not exactly. I don't want to have the file auto download. Instead, I'll have examples raise an exception if mnist.h5 does not exist that specifies that it can be obtained by running the mist/download_mnist.py script.